app/views/flood_risk_engine/enrollments/addresses/edit.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= link_to(
```

Comment:
I've moved the header and form into partials. Errors were already in partials. So I'm a little confused as what else you are suggesting should be moved into a partial.